.github/workflows/_deploy_production.yml

```diff
@@ -0,0 +1,100 @@
+name: Deploy Production
+run-name: Triggered by ${{ github.actor }}
+on:
+  workflow_call:
+    inputs:
+      tag:
+        description: "Image tag to deploy. Defaults to the last commit SHA in the branch."
```

Comment:
I think we can use `gh` to fetch the latest Actions run for the given commit SHA and make sure the CI did pass.